src/app/mesh-overview/components/MeshCharts.vue

```diff
@@ -73,7 +82,7 @@ function loadData() {
 }
 
 .chart-1\/4 {
```

Comment:
**Suggestion**: Ought to rename this according to the change in ratio.

```suggestion
.chart-1\/5 {
```

Reply:
😆 ah defo yeah, thanks for the spot

I made a slightly different change that takes the value out of the class name, see last commit